### DISCUSSION TITLE
The command "php artisan schedule:run" does not exist.

### DISCUSSION THREAD

**Discussion by hakihosting** [laravel/framework#46503] [Q&A]
- Laravel Version: 9.0.
- PHP Version: #.#.#
- Database Driver & Version:

### Steps To Reproduce:

Hello all when i execute i have this error : The command "php artisan schedule:run" does not exist. 
Somoene can help me ?

**Comment by lk77** [comment]
Likely a shell issue, check that you have php available by doing `php -v`
